go.sum

```diff
@@ -1,144 +1,666 @@
+4d63.com/gochecknoglobals v0.0.0-20201008074935-acfc0b28355a h1:wFEQiK85fRsEVF0CRrPAos5LoAryUsIX1kPW/WrIqFw=
```

Comment:
This is a lot of extras that I can't reconcile. Can you try regenerating this after addressing the `go.mod` comment?